docs/UserGuide.md

```diff
@@ -29,9 +29,13 @@ Workbook is optimized for fast typists and utilizes a **Command Line Interface (
 
 3. Copy the file to a folder of your choice to store data from your WorkBook.
 
-4. Using your command terminal, navigate to the folder where you placed your WorkBook at and fire it up by inputting: `java -jar WorkBook.jar` in your terminal. <br> The GUI with sample data as shown below should appear in a few seconds.<br>
-   ![Ui](images/Ui.png)
+4. Using your command terminal, navigate to the folder where you placed your WorkBook at and fire it up by inputting: `java -jar WorkBook.jar` in your terminal. <br> The GUI with sample data should appear in a few seconds. Depending on the window width, you will see one of the two layouts shown below.<br>
 
+Narrow Layout            |  Wide Layout
+:-------------------------:|:-------------------------:
+![narrow layout](images/UiNarrow.png) |   ![wide layout](images/Ui.png)
+
+<br>
```

Comment:
prolly incorporate this under `#### Responsive UI when resizing window` then just have to label the narrow UI's lightbulb in the image because I've labelled the main one alr

Reply:
I've labelled the narrow UI's lightbulb button. However, in the getting started section, i added the non annotated images and I've moved the annotated images into the responsive UI section.